### PR TITLE
Entities

### PR DESCRIPTION
**Pull request by PaddyDoyle88**
Make sure you've checked off all these things before submitting:

- [ ] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [ ] This pull request is submitted to the `dev` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -560,7 +568,8 @@ echo $response->getBody();
       "delta_time": 0,
       "nw_score": 0,
       "max_nw_score": 0,
-      "topics": []
+      "topics": [],
+	  "entities": []
```
Comment (mathewcropper) [inline source/index.html.md]: Style: should this be aligned in the same way as `"topics"`?

File: source/index.html.md [hunk 2]
```diff
@@ -476,7 +476,15 @@ echo $response->getBody();
           "name": "Entertainment"
         }
       ],
-      "has_video": true
+      "has_video": true,
+	  "entities": [
```
Comment (mathewcropper) [inline source/index.html.md]: Style: should this be aligned in the same way as `"has_video"`?

**Review by mathewcropper — CHANGES_REQUESTED** [review]
Very, very minor feedback on alignment of the text. That's all.

Very good catch re: text saying Facebook in the Instagram section.

File: source/index.html.md (hunk 2 above)
Comment (PaddyDoyle88) [reply source/index.html.md]: It is aligned when on the actual site. It just looks odd there as github treats spaces vs tabs differently. I can change it if you want

Comment (mathewcropper) [reply source/index.html.md]: Nope, no need. If it looks right on the site, that's all that matters. I'll change to approved :)

**Review by mathewcropper — APPROVED** [review]

**Pull request merged**